CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing code
```

Comment:
This will override the dart-lang/ default at https://github.com/dart-lang/.github/blob/main/CONTRIBUTING.md (the UI will show this file instead of that one).

I would ether reference the fuller doc from here, or, remove this file and in-line the bit about enabling logs in the readme.